src/test/ui/imports/extern-crate-self-macro-item.rs

```diff
@@ -0,0 +1,5 @@
+macro_rules! accept_item { ($x:item) => {} }
```

Comment:
This should be a compile-pass test, the default for UI is compile-fail.

Also add a comment above...